allennlp/modules/token_embedders/embedding.py

```diff
@@ -66,7 +66,14 @@ class Embedding(TokenEmbedder):
         If given, this will scale gradients by the frequency of the words in the mini-batch.
     sparse : bool, (optional, default=False):
         Whether or not the Pytorch backend should use a sparse representation of the embedding weight.
-
+    vocab_namespace : str, (optional, default=None):
+        In case of fine-tuning/transfer learning, the model's embedding matrix needs to be
+        extended according to the size of extended-vocabulary. To be able to know how much to
+        extend the embedding-matrix, it's necessary to know which vocab_namspace was used to
+        construct it in the original training. We store vocab_namespace used during the original
+        training as an attribute, so that it can be retrieved during fine-tuning.
+        Caveat: This would prevent models trained before this commit to be unextendedable.
```

Comment:
I don't understand. I meant that earlier we didn't store `_vocab_namespace` as an attribute so if we want to extend an old model, there is no way to find original namespace used for `Embedding` objects. So it's not extendable (and won't be extendable even in future), unless the archived model's embedding uses default namespace 'tokens'.  I will shift Caveat to comment, but it should still be caveat an not TODO, right?

Reply:
I guess I was looking at the question below.  That question is definitely a TODO, or a PR comment.  Maybe it could just be removed.  We _could_ do crazy things to try to recover it (like look through the original config file), but that's not worth the effort and complexity.

Then the caveat can stay there, but you can't reference "this commit" in a docstring or a comment, because it doesn't mean anything.  Maybe: "Added in version 0.9; models trained with 0.8.1 or below need special treatment when extending their vocabularies."